phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -967,9 +967,14 @@ NUMBER
     :   POSINTEGER
     ;
 
-// Exponential format is not supported.
 DECIMAL
```

Comment:
I think it looks good. Not sure if there's an official syntax for parsing an E notation for SQL-92. This requires that the E notation be there, which is what we want. These are all valid then:

    .1 e 100
    .01 e + 100
    1. e -100
    1.23 e 200